frontend/index.js

```diff
@@ -0,0 +1,40 @@
+document.addEventListener("DOMContentLoaded", () => {
+  let addClassForm = document.querySelector(".addClass")
+  let addStudentForm = document.querySelector(".addStudent")
+  let listStudentsForm = document.querySelector(".listStudents")
+  let listStudentsInput = document.querySelector("#listStudentsInput") 
+  
+  addClassForm.addEventListener("submit", async (e) => {
+    e.preventDefault();
+    let addAClassLi = document.querySelector("#addAClass");
+    let classNameInput = document.querySelector("#classNameInput") ;
+    let teacherNameInput = document.querySelector("#teacherNameInput");
+    // let newClass = [];
+
+    // newClass.push({name: classNameInput.value, teacher: teacherNameInput.value})
+    await axios.post("http://localhost:3000/class/", {name: classNameInput.value, teacher: teacherNameInput.value}) 
+    
+    addAClassLi.innerHTML = "";
+    addAClassLi.innerText = "You've created a class " + {name: classNameInput.value, teacher: teacherNameInput.value};
```

Comment:
This is very deceiving. You are creating an illusion that everything went through when it didn't. The created class should come from your response